jaws-journey-deploy/README.md

```diff
@@ -18,6 +18,8 @@ Available Jobs
 * [notify-shipit](#notify-shipit)
 * [run-automation-test](#run-automation-test)
 
+All the available jobs apart from checkout-code will notify of failure via a slack message.  For more info go to [slack-notification](#slack-notification) section
```

Comment:
Should maybe say only in prod and nonprod?

Reply:
technically it could be any environment - it is dependant on what setting SLACK_INTEGRATION_ENABLED has

Reply:
I should really read the whole thing before I post